_ml-commons-plugin/api/mcp-server-apis/sse-session.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: SSE Session
+parent: Agent APIs
+grand_parent: ML Commons APIs
+nav_order: 50
+---
+
+# SSE Session
+**Introduced 3.0**
+{: .label .label-purple }
+
+This is the standard sse sessoin creation API of MCP protocol, you can use this API to create a SSE session between client and MCP server in OpenSearch. Usually you don't need to explicitly interact with this API if you're using standard MCP client. You can checkout the example client here: https://github.com/zane-neo/opensearch-mcpserver-test-example
```

Comment:
Same with this example: can we host it in the ML Commons repo?

Reply:
It's not possible that in this time we can make it in ml commons repo, I can remove it if it's not a good approach.